services/callback/src/test/java/app/coronawarn/server/services/callback/controller/ApiExceptionHandlerTest.java

```diff
@@ -2,6 +2,7 @@
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.lenient;
```

Comment:
What does this do?

Reply:
This was used to mitigate the doThrow(RuntimeException.class).when(serviceMock).save(any()) exception. 

"Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at app.coronawarn.server.services.callback.controller.ApiExceptionHandlerTest.testUnexpectedExceptionsTriggersStatusCode500(ApiExceptionHandlerTest.java:38)
Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class."